### PR TITLE
Implement upload annotation

### PR DESCRIPTION
**Pull request by haoxusci**
approved with test. but wait to merge until backend implement uploadannotation

### CODE REVIEW

File: src/SingleCellAnnotator.imjoy.html [hunk 1]
```diff
@@ -413,9 +417,11 @@
       // this is the url to send the current annotation
       const sub_url = "annotations/" + this.current_task_name + "/" + this.current_image_data.cell_id;
       const url_post = this.selected_server_url + sub_url;
+      api.alert(url_post)
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: Are you going to keep this one?

File: src/SingleCellAnnotator.imjoy.html [hunk 2]
```diff
@@ -473,6 +479,15 @@
       });
       api.alert(JSON.stringify(blob))
     },
+    // go to nextImage after select yes/no.
+    async singleSelected(){
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: Perhaps a better name, say, `selectLocation(true/false)` with a parameter for selected or not so we don't need two functions for this.

Comment (oeway) [reply src/SingleCellAnnotator.imjoy.html]: Also there is another place with the Y/N keybinding you need to change.

File: src/SingleCellAnnotator.imjoy.html (hunk 1 above)
Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: now removed

File: src/SingleCellAnnotator.imjoy.html (hunk 2 above)
Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: added

**Comment by oeway** [comment]
Please integrate changes in #28

**Comment by haoxusci** [comment]
the features were merged. with uploadannotation muted

**Pull request closed without merging**